### PR TITLE
chore(resin): Add resin tags to annotation activity

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js [hunk 1]
```diff
@@ -245,7 +246,12 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
-            <div className="bcs-activity-feed" data-testid="activityfeed" onKeyDown={this.onKeyDown}>
+            <div
+                className="bcs-activity-feed"
+                data-resin-component={ACTIVITY_TARGETS.ACTIVITY_FEED}
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:251]: The `data-resin-component` for the VersionsSidebar is `preview`. Should we remain consistent between sibling sidebar components?

File: src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ const AnnotationActivity = ({
         : { ...linkMessage, values: { number: linkValue } };
 
     return (
-        <div className="bcs-AnnotationActivity">
+        <div className="bcs-AnnotationActivity" data-resin-feature="annotations">
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js:64]: This is not annotations, though, it's activity feed.

File: src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ const AnnotationActivityLink = ({
     return (
         <PlainButton
             className="bcs-AnnotationActivity-link"
+            data-resin-target="markupLink"
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx:33]: Should this be `annotationLink` or something similar?

File: src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js (hunk 2 above)
Comment (mxiao6) [reply src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js:64]: All of the annotations related resin tags have `feature="annotations"` I think it's just easy for Rohit to collect annotations logs.

**Pull request merged**